schema.sql

```diff
@@ -21,7 +21,8 @@ CREATE UNIQUE INDEX clients_data_pkey ON clients(id int4_ops);
 
 CREATE TABLE clients_weights (
     id SERIAL PRIMARY KEY,
-    weight integer NOT NULL,
+    weight numeric(10,2)
+ NOT NULL,
```

Comment:
this is a db schema change

Reply:
you'll need to update the schema rather than re-creating this table

Reply:
done